build.rs

```diff
@@ -0,0 +1,32 @@
+extern crate winres;
+use winres::VersionInfo;
+
+fn main() {
+    // only run if target os is windows
+    if std::env::var("CARGO_CFG_TARGET_OS").unwrap() != "windows" {
+        println!(
+            "cargo:warning={:#?}",
+            "This build script is only for windows target, skipping..."
+        );
+        return;
+    }
+
+    let mut res = winres::WindowsResource::new();
+
+    //version  X. X. X. X
+    //         ⇑  ⇑  ⇑
+    //         48 32 16
+    let mut version: u64 = 0;
+    version |= 0 << 48;
```

Comment:
better use here package info from env variables:
- `CARGO_PKG_VERSION` — The full version of your package.
- `CARGO_PKG_VERSION_MAJOR` — The major version of your package.
- `CARGO_PKG_VERSION_MINOR` — The minor version of your package.
- `CARGO_PKG_VERSION_PATCH` — The patch version of your package.